testutil/kubernetes_helper.go

```diff
@@ -281,6 +281,15 @@ func (h *KubernetesHelper) CheckService(ctx context.Context, namespace string, s
 	})
 }
 
+// GetService gets a service that exists in a namespace.
+func (h *KubernetesHelper) GetService(ctx context.Context, namespace string, serviceName string) (*corev1.Service, error) {
```

Comment:
This is the same amount of indirection as `CheckIfNamespaceExists` and `GetSecret`. The main thing here is that `clientset` is not public, so it would require changing that. I'm not sure changing that and removing those helpers as well should be coupled with this PR, but maybe it makes sense in a separate change.

Reply:
ah, I didn't notice that `clientset` was not public.  Makes sense!